docs/user_peripherals/22_spike.md

```diff
@@ -0,0 +1,49 @@
+<!---
+
+This file is used to generate your project datasheet. Please fill in the information below and delete any unused
+sections.
+
+You can also include images in this folder and reference them in the markdown. Each image must be less than
+512 KB in size, and the combined size of all images must be less than 1 MB.
+-->
+
+## What it does
```

Comment:
Please add a section above here similar to https://github.com/TinyTapeout/ttsky25a-tinyQV/blob/main/docs/user_peripherals/21_matt_pwm.md?plain=1#L14-L19

(this was added to the template after you forked).